src/cljs/rems/profile.cljs

```diff
@@ -95,12 +121,47 @@
                   [:button.btn.btn-primary
                    {:type "submit"}
                    (text :t.profile/save)]])}]
+
      [:div.mt-3
       [collapsible/component
        {:title (text :t.profile/your-details)
         :always [:<>
                  [user/username (:user identity)]
-                 [user/attributes attributes false]]}]]]))
+                 [user/attributes (:user identity) false]]}]]
+
+     (when (and (:enable-ega config) (roles/has-roles? :handler))
```

Comment:
this could be a named component as well

Reply:
I refactored the whole page component so that it can stay at one abstraction level.